### ISSUE TITLE
Clone the project and run, but it show empty of all chars

### ISSUE THREAD

**Issue by parcool** [AnyChart/AnyChart-Android#223]
I just clone the project and run, what did I wrong?

https://user-images.githubusercontent.com/12092478/124549666-11b26f80-de62-11eb-90b3-8de402cc485f.mp4

**Comment by parthitechnotion** [comment]
Same issue I am also facing.
Please provide solution as I have already used this in our project

**Comment by Shestac92** [comment]
@parcool @parthitechnotion 
Please, check the [Quick Start guide](https://github.com/AnyChart/AnyChart-Android/wiki/Getting-started), it describes how to launch the framework in your projects.

**Comment by vasiledoe** [comment]
@Shestac92 nope, I got same empty  result following the  Quick Start guide. Running Android 11, and v1.1.4
`    implementation 'com.github.AnyChart:AnyChart-Android:1.1.4'`

**Comment by Sicness** [comment]
I have the same problem with the latest Android Studio. Just cloned the project and run the sample module from there. The sample In the repo doesn't work. API29.

**Comment by pinefirst** [comment]
I just cloned and build but empty. My sdk compile verion is 31.
Please help me...

**Comment by viniciushrk** [comment]
Verify if code append data in AnyChart.pie();
example:

```
            Pie pie = AnyChart.pie();
            List<DataEntry> data = new ArrayList<>();
            data.add(new ValueDataEntry("John", 10000));
            data.add(new ValueDataEntry("Jake", 12000));
            data.add(new ValueDataEntry("Peter", 18000));
            pie.data(data);
            AnyChartView anyChartView = (AnyChartView) findViewById(R.id.graph);
            anyChartView.setChart(pie);
```
